tests/components/unifi/test_hub.py

```diff
@@ -167,6 +169,11 @@ def mock_default_unifi_requests(
         json={"data": wlans_response or [], "meta": {"rc": "ok"}},
         headers={"content-type": CONTENT_TYPE_JSON},
     )
+    aioclient_mock.put(
```

Comment:
I think this fits better with the specific test. This function is mostly to make the integration setup easier

Reply:
Ok